include/blas_meta.h

```diff
@@ -222,6 +222,18 @@ struct is_complex_std
 
 #endif
 
+class unsupported_exception : public std::runtime_error {
+ public:
+  unsupported_exception(const std::string &operator_name)
+      : std::runtime_error(operator_name), _msg(operator_name) {
```

Comment:
I don't like that we're storing the operator name unused in the parent class, but all `runtime_error` constructors seem to require this. Is there anything we can do to avoid this?

Reply:
I checked the `std::runtime_error` class from [here](https://en.cppreference.com/w/cpp/error/runtime_error) and all constructors require a string or another exception as argument, so I don't think we can avoid it.